internal/test_helpers/fixtures/bash/history_persistence/pass

```diff
@@ -0,0 +1,162 @@
+Debug = true
+
+[33m[stage-5] [0m[94mRunning tests for Stage #5: za2[0m
+[33m[stage-5] [0m[94mRunning ./your_shell.sh[0m
+[33m[stage-5] [0m[94mHistory file content:[0m
+[33m[stage-5] [0m[pear_shell_history_test] echo blueberry mango
+[33m[stage-5] [0m[pear_shell_history_test] echo pineapple grape
+[33m[stage-5] [0m[pear_shell_history_test] echo grape apple
+[33m[your-program] [0m$ echo mango orange pear
+[33m[your-program] [0mmango orange pear
+[33m[stage-5] [0m[92m✓ Ran echo mango orange pear[0m
+[33m[your-program] [0m$ echo raspberry banana
+[33m[your-program] [0mraspberry banana
+[33m[stage-5] [0m[92m✓ Ran echo raspberry banana[0m
+[33m[your-program] [0m$ echo raspberry strawberry
+[33m[your-program] [0mraspberry strawberry
+[33m[stage-5] [0m[92m✓ Ran echo raspberry strawberry[0m
+[33m[your-program] [0m$ history
+[33m[your-program] [0m    1  echo mango orange pear
+[33m[your-program] [0m    2  echo raspberry banana
+[33m[your-program] [0m    3  echo raspberry strawberry
+[33m[your-program] [0m    4  history
+[33m[stage-5] [0m[92m✓ History before loading file is correct[0m
+[33m[your-program] [0m$ history -r /tmp/pear_shell_history_test
```

Comment:
There's a lot of setup here before we actually get to testing `history -r`. 3 commands in the file, 3 commands in-memory... Can we simplify and just run `history -r` as the first command? Users are going to see failures when they start this stage, the less confusing it is the better.